R/Filter.R

```diff
@@ -619,6 +619,254 @@ makeFilter(
   }
 )
 
+# Preprocess the data before applying praznik filter. According to Praznik documentation:
+# The best practice is to convert data to factors prior to feeding them in this function.
+# Real attributes are cut into about 10 equally-spaced bins, following the heuristic often used in literature.
+# Precise number of cuts depends on the number of objects; namely, it is n/3, but never less than 2 and never more than 10.
+preprocess.cmi.praznik = function(data) {
+    colns = colnames(data)
+    data = convertDataFrameCols(data, logicals.as.factor = TRUE, chars.as.factor = TRUE)
+    int.yes = vlapply(data, is.integer)
```

Comment:
Do we really want this? I think we should treat them like numeric values?

Reply:
Hi, do you mean whether we should split the data rather than keep the continuous values?  preprocess.cim.praznik is optional,  which we could switch off, but the author said in the docs that it is better to discretize continous values before hand. see praznik doc. what do you think? @mllg